src/volatile-root/volatile-root.c

```diff
@@ -175,6 +175,8 @@ static int run(int argc, char *argv[]) {
         r = get_block_device_harder(path, &devt);
         if (r < 0)
                 return log_error_errno(r, "Failed to determine device major/minor of %s: %m", path);
+        else if (r == 0)
+                return log_error_errno(r, "Path %s has no underlying block device, ignoring.", path);
```

Comment:
Oh, please use `log_debug()`. `_errno` is not necessary here as `r` is zero, and I think we do not need to log as a error.